From 9f0906813ff54d4fb4aacb5c33e0eb657609620b Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 31 Jul 2016 15:34:09 +0300
Subject: [PATCH 0648/2241] Documentation: miscdevice: mvebu: add devices for
 marvell telephony drivers

Add 2 miscdevices for Marvell telephony drivers:
- SLIC device - might have different names for different manufacturers
- TAL device - user-space interface for telephony tool

Change-Id: I7926a7a914f382ef256eaac2cf9d5bb9f01956cc
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31565
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
---
 Documentation/devices.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devices.txt b/Documentation/devices.txt
index 87b4c5e..bc18808 100644
--- a/Documentation/devices.txt
+++ b/Documentation/devices.txt
@@ -345,6 +345,10 @@ Your cooperation is appreciated.
 		 13 = /dev/vpcmouse	Connectix Virtual PC Mouse
 		 14 = /dev/touchscreen/ucb1x00  UCB 1x00 touchscreen
 		 15 = /dev/touchscreen/mk712	MK712 touchscreen
+		 73	= /dev/silabs	Marvell silabs SLIC device
+		 	  or			or
+		 	  /dev/vpapi	Marvell vpapi SLIC device
+		 74 = /dev/tal		Marvell tal device
 		128 = /dev/beep		Fancy beep device
 		129 =
 		130 = /dev/watchdog	Watchdog timer port
-- 
2.7.4

